.env.example

```diff
@@ -32,3 +32,8 @@ TEST_DISCORD_WEBHOOK=
 HCAPTCHA_SECRET=
 HCAPTCHA_SITEKEY=
 COORDINAPE_USER_ADDRESS=0xfad763da9051953fea58c2304395719a3b6ba361
+
```

Comment:
can we just add a comment saying `# set these to true to enable, false to disable` since some of these may be amgiguous